torch/ao/quantization/experimental/apot_utils.py

```diff
@@ -8,7 +8,13 @@
 r"""Converts floating point input into APoT number
     based on quantization levels
 """
-def float_to_apot(x, levels, indices):
+def float_to_apot(x, levels, indices, alpha):
```

Comment:
can we just use torch.clamp?

Reply:
@dzdang I'm not using that here since x is a float value, not a tensor

Reply:
i see. thought there'd be an overload that took in a float